br/pkg/task/backup_ebs.go

```diff
@@ -253,6 +273,123 @@ func RunBackupEBS(c context.Context, g glue.Glue, cmdName string, cfg *BackupEBS
 	return nil
 }
 
+func waitAllScheduleStoppedAndNoRegionHole(ctx context.Context, cfg Config, mgr *conn.Mgr) error {
+	allStores, err := conn.GetAllTiKVStoresWithRetry(ctx, mgr.GetPDClient(), util.SkipTiFlash)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	// we wait for 15*40 = 600s = 10m at least
+	for i := 0; i < 40; i++ {
+		if ctx.Err() != nil {
+			return ctx.Err()
+		}
+		if i > 0 {
+			time.Sleep(waitAllScheduleStoppedInterval)
+		}
+		allRegions, err2 := waitUntilAllScheduleStopped(ctx, cfg, allStores, mgr)
+		if err2 != nil {
+			if causeErr := errors.Cause(err2); causeErr == hasPendingAdminErr {
+				log.Info("schedule ongoing on tikv, will retry later", zap.Error(err2))
+			} else {
+				log.Warn("failed to wait schedule, will retry later", zap.Error(err2))
+			}
+			continue
+		}
+
+		// sort by start key
+		sort.Slice(allRegions, func(i, j int) bool {
+			left, right := allRegions[i], allRegions[j]
+			return bytes.Compare(left.StartKey, right.StartKey) < 0
+		})
+
+		var hasHole bool
+		for j := 0; j < len(allRegions)-1; j++ {
+			left, right := allRegions[i], allRegions[j]
```

Comment:
what is 'i' of allRegions[I]? maybe 'i' does not start from 0?

Reply:
it's a bug